README.md

```diff
@@ -12,11 +12,27 @@ npm install --save-dev broccoli-merge-trees
 ```
 
 ## Usage
+1. With `new`
+```js
+const BroccoliMergeTrees = require('broccoli-merge-trees');
 
+let mergedNode = new BroccoliMergeTrees(inputNodes, options);
+```
+2. As a `function call`
 ```js
-var BroccoliMergeTrees = require('broccoli-merge-trees');
+const broccoliMergeTrees = require('broccoli-merge-trees');
+
+let mergedNode = broccoliMergeTrees(inputNodes, options);
+```
+
+3. As a `base class`
```

Comment:
We do not support folks extending from our Plugin base class. We expect to only be used directly for the functionality we provide.